app/services/support_interface/notifications_export.rb

```diff
@@ -0,0 +1,50 @@
+module SupportInterface
+  class NotificationsExport
+    def data_for_export
+      courses = Course.where(recruitment_cycle_year: RecruitmentCycle.current_year, open_on_apply: true)
+      providers = Provider.includes(:provider_users, :application_choices).joins(:courses).merge(courses).group('providers.id')
+      providers.flat_map do |provider|
+        data_for_provider(provider, provider_users_with_make_decisions(provider), visible_applications(provider))
+      end
+    end
+
+  private
+
+    def data_for_provider(provider, users_with_make_decisions, applications)
+      {
+        'Provider Code' => provider.code,
+        'Provider' => provider.name,
+        'No. Applications Received' => applications.count,
+        'No. Applications awaiting decisions' => applications.count(&:awaiting_provider_decision?),
```

Comment:
Shouldn't 'applications awaiting decisions' include applications at the `interviewing` state?

Reply:
Good spot - I've modified it to use the awaiting decisions grouping instead, and I've added a couple of extra application choices to the test which help tease out the differences